{{cookiecutter.project_slug}}/clients/vue3/tailwind.config.js

```diff
@@ -0,0 +1,28 @@
+/** @type {import('tailwindcss').Config} */
+module.exports = {
+  content: ['./index.html', './src/**/*.{vue,js}'],
+  theme: {
+    extend: {
+      colors: {
+        primary: '#042642',
+        accent: '#d93a00',
```

Comment:
**Info:** Set `primary` and `accent` colors that match ThinkNimble's colors, but they can be easily adjusted for each project as needed.

Reply:
Love this, great way to demonstrate how to build a theme in tailwind